### PR TITLE
feat(@clayui/empty-state): Include Small property in empty state component

### PR DESCRIPTION
**Pull request by sergiojimcos**
**What is new?**
New feature in clayui/empty-state, `small` property is included in this component.

https://user-images.githubusercontent.com/44723449/153244299-eef1558b-1d40-4804-b462-190904464f9f.mov

**Fixes**
#4641

### CODE REVIEW

**Comment by julien** [comment]
@matuzalemsteles just so you know, I had to edit our `format:changed` script because it wasn't checking scss files. @sergiojimcos sorry about that.

File: packages/clay-empty-state/src/index.tsx [hunk 1]
```diff
@@ -42,15 +48,26 @@ const ClayEmptyState: React.FunctionComponent<IProps> = ({
...
-				<div className="c-empty-state-image">
+				<div
+					className={classNames(
+						className,
+						{
+							'c-empty-state-image': !small,
+						},
+						{
+							'c-empty-state-image-sm': small,
+						}
```
Comment (matuzalemsteles) [inline packages/clay-empty-state/src/index.tsx:65]: This is just a small change 😁 but in these cases, you can group everything into one object, it wouldn't be necessary to separate each condition into an object.

**Review by matuzalemsteles — COMMENTED** [review]
Well looks good too, good job! I just left a comment for adjustment but very small.

File: packages/clay-empty-state/src/index.tsx (hunk 1 above)
Comment (sergiojimcos) [reply packages/clay-empty-state/src/index.tsx:65]: You are right, I haven't seen that before 😄

**Comment by pat270** [comment]
@sergiojimcos Thanks for the pull. I think one improvement we can make is adding the sm class at the top dom element. It would be `c-empty-state-sm`. The Sass would look like:

```scss
$c-empty-state-sm: () !default;
$c-empty-state-sm: map-deep-merge(
  (
    c-empty-state-animation: (),
    c-empty-state-image: (),
    c-empty-state-title: (),
  ),
  $c-empty-state-sm
);

.c-empty-state-sm {
  &.c-empty-state-animation {
    @include clay-css(map-get($c-empty-state-sm, c-empty-state-animation));
  }

  .c-empty-state-image {
    @include clay-css(map-get($c-empty-state-sm, c-empty-state-image));
  }
}
```

File: packages/clay-css/src/scss/variables/_empty-state.scss [hunk 2]
```diff
@@ -13,6 +13,19 @@ $c-empty-state-animation: map-deep-merge(
 	$c-empty-state-animation
 );
 
+$c-empty-state-animation-sm: () !default;
+$c-empty-state-animation-sm: map-deep-merge(
+	(
+		align-items: center,
```
Comment (pat270) [inline packages/clay-css/src/scss/variables/_empty-state.scss:19]: A lot of styles are duplicated, we can just inherit them from the original component.

**Comment by julien** [comment]
@sergiojimcos I'll have another look, but for the CSS part, let's ping @pat270

File: package.json [hunk 3]
```diff
@@ -22,7 +22,7 @@
 		"coverage": "jest --coverage",
 		"format": "prettier --write \"**/*.{js,ts,tsx,md,mdx,json,scss}\"",
 		"format:all": "yarn lint:fix && yarn format",
-		"format:changed": "git ls-files -mz \"*.js\" \"**/*.js\" \"**/*.ts\" \"**/*.tsx\" \"*.md\" \"**/*.md\" \"**/*.mdx\" \"*.json\" \"**/*.json\" | xargs -0 prettier --write --",
+		"format:changed": "git ls-files -mz \"*.js\" \"**/*.js\" \"**/*.ts\" \"**/*.tsx\" \"*.md\" \"**/*.md\" \"**/*.mdx\" \"*.json\" \"**/*.json\" \"**/*.scss\" | xargs -0 prettier --write --",
```
Comment (julien) [inline package.json:25]: Good catch.

File: packages/clay-css/src/scss/variables/_empty-state.scss (hunk 2 above)
Comment (julien) [reply packages/clay-css/src/scss/variables/_empty-state.scss]: @pat270 could you please have another look, it seems @sergiojimcos updated this. Thanks!

**Comment by julien** [comment]
@pat270 thanks again

**Comment by sergiojimcos** [comment]
> I updated the scss. This should be gtg.

Thanks!!

**Pull request merged**